src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -172,7 +172,7 @@ public DashboardWebApplication(
                 // This is added so CORS middleware doesn't report failure for dashboard browser requests that include an origin header.
                 options.AddDefaultPolicy(builder =>
                 {
-                    builder.WithOrigins(dashboardOptions.Frontend.GetEndpointUris().Select(uri => uri.OriginalString).ToArray());
+                    //builder.WithOrigins(dashboardOptions.Frontend.GetEndpointUris().Select(uri => uri.OriginalString).ToArray());
```

Comment:
?

Reply:
Fixing it now. This was busted before if there was a dynamic IP, although the side effect was just some warnings.